.github/workflows/bf-review.yml

```diff
@@ -4,6 +4,8 @@ on:
     paths:
     - 'backend/helsinki-benefit/**'
     - 'frontend/helsinki-benefit/**'
+    - 'frontend/shared/**'
+    - 'frontend/*'
```

Comment:
Hmn... Wouldn't this will trigger the Helsinki Benefit build when there is a change in `frontend/kesaseteli` ?

Reply:
as far as I understand it should not include subfolders: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
Ah, yeah, now I get it :) Thanks for the clarification